app/app.js

```diff
@@ -187,24 +190,28 @@ function processHeroProfile(){
       profileDescription.html('<p>'+heroDesc+'</p>');
       profileRealName.html(heroSecretId)
 
+      // Give the profile header the proper images
+      profileHeader.background = 'url(' + heroBackground + ')no-repeat center center / cover';
```

Comment:
Maybe we could put the `background-repeat`, `background-position` definitions in the CSS and only override the `background-image` here? Seems more clean